test/postman/inference_data.json

```diff
@@ -1,78 +1,182 @@
-		"model_name": "alexnet",
-		"worker": 1,
-		"synchronous": "true",
-		"file": "../examples/image_classifier/kitten.jpg"
-	},
-	{
-		"url": "https://torchserve.s3.amazonaws.com/mar_files/resnet-18.mar",
-		"model_name": "resnet-18",
-		"worker": 1,
-		"synchronous": "true",
-		"file": "../examples/image_classifier/kitten.jpg"
-	},
-	{
-		"url": "https://torchserve.s3.amazonaws.com/mar_files/vgg11.mar",
-		"model_name": "vgg11",
-		"worker": 1,
-		"synchronous": "true",
-		"file": "../examples/image_classifier/kitten.jpg"
-	},
-	{
-		"url": "https://torchserve.s3.amazonaws.com/mar_files/mnist.mar",
-		"model_name": "mnist",
-		"worker": 1,
-		"synchronous": "true",
-		"file": "../examples/image_classifier/mnist/test_data/0.png"
-	},
-	{
-		"url": "https://torchserve.s3.amazonaws.com/mar_files/fastrcnn.mar",
-		"model_name": "fastrcnn",
-		"worker": 1,
-		"synchronous": "true",
-		"file": "../examples/object_detector/persons.jpg"
-	},
-	{
-		"url": "https://torchserve.s3.amazonaws.com/mar_files/maskrcnn.mar",
-		"model_name": "maskrcnn",
-		"worker": 1,
-		"synchronous": "true",
-		"file": "../examples/object_detector/persons.jpg"
-	},
-	{
-		"url": "https://torchserve.s3.amazonaws.com/mar_files/fcn_resnet_101.mar",
-		"model_name": "fcn_resnet_101",
-		"worker": 1,
-		"synchronous": "true",
-		"file": "../examples/object_detector/persons.jpg"
-	},
-	{
-		"url": "https://torchserve.s3.amazonaws.com/mar_files/resnet-152-batch.mar",
-		"model_name": "resnet152",
-		"worker": 1,
-		"synchronous": "true",
-		"file": "../examples/image_classifier/kitten.jpg"
-	}
+[
+    {
+        "url":"https://torchserve.s3.amazonaws.com/mar_files/my_text_classifier.mar",
+        "model_name":"my_text_classifier",
+        "worker":1,
+        "synchronous":"true",
+        "file":"../examples/text_classification/sample_text.txt",
+        "output_type":"text",
+        "expected":"Business"
+    },
+    {
+        "url":"https://torchserve.s3.amazonaws.com/mar_files/squeezenet1_1.mar",
+        "model_name":"squeezenet1_1",
+        "worker":1,
+        "synchronous":"true",
+        "file":"../examples/image_classifier/kitten.jpg",
+        "output_type":"json",
+        "expected":[
+            {
+                "tabby":0.2752002477645874
+            },
+            {
+                "lynx":0.2546876072883606
+            },
+            {
+                "tiger_cat":0.24254210293293
+            },
+            {
+                "Egyptian_cat":0.2213735282421112
+            },
+            {
+                "cougar":0.0022544863168150187
+            }
+        ],
+        "tolerance":1
+    },
+    {
+        "url":"https://torchserve.s3.amazonaws.com/mar_files/densenet161.mar",
+        "model_name":"densenet161",
+        "worker":1,
+        "synchronous":"true",
+        "file":"../examples/image_classifier/kitten.jpg",
+        "output_type":"json",
+        "expected":[
+            {
+                "tiger_cat":0.4693359136581421
```

Comment:
@shivamshriwas The model will never return the same exact prediction scores on each attempt - there will be a slight variation. So these checks will fail. A better check will be to just search for the names in the resuling json like tiger_cat. Same comment for all the other 'expected' values that added in this PR

Reply:
@chauhang To solve this problem,  "tolerance" parameter has been added to this expected response template [see line 61 below]. However it seems there is not enough documentation around response template specification. @shivamshriwas Can you please add a description around response template parameters.

Reply:
@chauhang These checks will not fail as we are comparing the scores with a tolerance factor in consideration. currently it is set to 1%  tolerance. So scores can be +/- 1% of expected value.
This tolerance value can be tuned if required, but 1% seems to be fine tested the suite with multiple consecutive runs on cpu as well as gpu based ec2

Reply:
Thanks @shivamshriwas Please document the details as suggested by @dhaniram-kshirsagar

Reply:
@chauhang I I have added required documentation.